src/pages/ViewSoulKeyPage/ViewSoulKeyPageContents.tsx

```diff
@@ -88,7 +88,12 @@ const PageContents = ({
           operationName: 'updateMyRepScore',
         }
       );
-      queryClient.invalidateQueries(['soulKeys', subjectAddress, 'profile']);
+      queryClient.invalidateQueries([
```

Comment:
So if we have a nice clear hierarcy of caching keys, then we can invalidate the entire tree of caches with only one invalidation, like 

```
queryClient.invalidateQueries([QUERY_KEY_SOUL_KEYS,  address]
```